controllers/database/constants.go

```diff
@@ -1,6 +1,31 @@
 package database
 
 const (
-	CheckDatabaseHealthError = "fail to check database"
-	CreateDatabaseCrError    = "fail to create inCluster database"
+	CheckDatabaseHealthError          = "Check database health error"
+	CreateDatabaseCrError             = "Create database CR error"
+	UpdateDatabaseCrError             = "Update database CR error"
+	GenerateRedisCrError              = "Generate database CR error"
+	SetOwnerReferenceError            = "Set owner reference error"
+	DefaultUnstructuredConverterError = "Default unstructured converter error"
+)
+
+const (
+	DownScalingDatabase     = "DatabaseDownScaling"
+	UpScalingDatabase       = "DatabaseUpScaling"
+	RollingUpgradesDatabase = "DatabaseRollingUpgrades"
+
+	MessageDatabaseCreate = "Database  %s already created."
+
+	MessageDatabaseUpdate = "Database  %s already update."
+
+	MessageDatabaseDownScaling     = "Database downscale from %d to %d"
+	MessageDatabaseUpScaling       = "Database upscale from %d to %d"
+	MessageDatabaseRollingUpgrades = "Database resource from %s to %s"
+)
+
+const (
+	InClusterDatabasePort        = "5432"
+	InClusterDatabaseUserName    = "postgres"
+	InClusterDatabaseName        = "postgres"
```

Comment:
is this database used for harbor-cluster?  Calling it `harbor-cluster` or something else instead of `postgres` will be easier to understand

Reply:
The harbor-core document configuration also uses the default database `postgres`, maybe we can use the default database to complete build harbor-core.
```shell
kubectl create secret "$COMPONENT-database" \
   --from-literal host="$COMPONENT-database-postgresql" \
   --from-literal port='5432' \
   --from-literal database='postgres' \
   --from-literal username='postgres' \
   --from-literal password="$PG_PASSWORD"
```

Reply:
If it's not a big effort, let's use a meaningful database name. The default db name we used in the online/offline installation is `registry`. Maybe `harbor-registry-db` is better?

Reply:
OK, I will commit a PR to fix this comment.